Lombiq.NodeJs.Extensions/config/consumer/package.project.json

```diff
@@ -4,13 +4,5 @@
     "build": "npm explore nodejs-extensions -- pnpm build",
     "clean": "npm explore nodejs-extensions -- pnpm clean",
     "watch": "npm explore nodejs-extensions -- pnpm watch"
-  },
-  "devDependencies": {
-    "eslint": "^8.25.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-only-warn": "^1.0.3",
-    "eslint-plugin-promise": "^6.1.0"
```

Comment:
Removing these is a good idea. I suggest removing it from _package.global.json_ as well, and reading the entries from NE's own _package.json_ file with a filter on `name.startsWith('eslint')`.